tests/integration/targets/snap/tasks/main.yml

```diff
@@ -35,18 +53,36 @@
       assert:
         that:
           - install is changed
+          - install_check is changed
           - install_again is not changed
+          - install_again_check is not changed
 
-    - name: check package has been installed correctly
+    - name: Check package has been installed correctly
       command: hello-world
+      environment:
+        PATH: /var/lib/snapd/snap/bin/
```

Comment:
I guess this should be handled in module itself, whether to seek `snap` binary itself or have an option which user can set.
Either way this can be added later in another PR.

Reply:
Which module do you mean? The `command` module?

Reply:
@aminvakil I also did not understand what you meant.

Reply:
Oops, I thought this belonged to snap module, never mind.